environments/builder/client/client.go

```diff
@@ -0,0 +1,28 @@
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+
+	"github.com/fission/fission"
+)
+
+func DoBuildRequest(builderUrl string, br *fission.PackageBuildRequest) error {
```

Comment:
I know it's just one function, but can you structure this like the other clients? (We may need to add other methods in the future, as we add stuff like progress reporting etc.)

i.e. a BuilderClient struct with a URL inside it, and a `build(br *fission.PackageBuildRequest)` method.